src/hooks/form.js

```diff
@@ -6,6 +6,7 @@ const useForm = (callback, defaultValues={}) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
+    console.log('values',values)
     callback(values);
```

Comment:
You're "leaking" state here, so modifications in the `addItem` callback are seen here, too.

Suggest either:

1. Pass out a copy of `values`
    ```suggestion
        callback({ ...values });
    ```
2. "Release" `values` to the callback and reset the hook state
    ```suggestion
        callback(values);
        setValues({});
    ```